lisa/tools/windows_feature.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any, List
+
+from lisa.executable import Tool
+from lisa.tools.powershell import PowerShell
+from lisa.util import LisaException
+
+
+# WindowsFeature management tool for Windows Servers.
+# It can install, uninstall, and check the status of Windows features.
+# Hyper-V, DHCP etc. are examples of Windows features.
+# This tool uses PowerShell to manage Windows features.
+# Not supported on PC versions like Windows 10, 11 etc.
+class WindowsFeature(Tool):
```

Comment:
`WindowsFeatureInstaller`

Reply:
This is for managing the Windows Features not just for installing. Simple WindowsFeature name suits better.

Reply:
Then call it `WindowsFeatureManagement`

Reply:
Fixed